lib/assessment.rb

```diff
@@ -5,35 +5,48 @@
 # Create a class called Dog. It should have, as attributes, name, breed, and age.
 # Make all of those attributes read-only.
 
-##
-# your code here
-##
+class Dog
+  def initialize(name, breed, age)
+    @name = name
+    @breed = breed
+    @age = age
+  end
+end
 
 # Question 2
 # Create a new Dog object.
 
-Answer.dog = nil # replace nil with your code
+Answer.dog = Dog.new("Kevin", "Golden-Doodle Husky", 2)
 
 # Question 3
 # Create a Person class, with attributes name, age, and location.
 # It should set all three attributes when you create a new Person.
 # Name and location should be writeable, but age should be read-only.
 
-##
-# your code here
-##
+class Person
+  attr_writer :name, :location
```

Comment:
Missed this mistake earlier - this should actually be `attr_accessor`, not `attr_writer`. `attr_writer` is almost never used because it only confers write access, _not read access_, and needing to use that particular combination is extremely uncommon.